sdk/python/tests/run_tests.sh

```diff
@@ -13,5 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cd $(dirname $0)
-python3 -m unittest discover --verbose --start-dir tests --top-level-directory=..
+cd $(dirname $0)/../kfmd
```

Comment:
Curious why not 
cd $(dirname $0)
python3 -m unittest discover --verbose --start-dir . --top-level-directory=..
?

Reply:
SG